host/tests/L3_TestCases/dsCompositeIn/dsCompositeInHelperClass.py

```diff
@@ -78,20 +80,16 @@ def testDownloadAssets(self):
 
         # List of streams with path
         self.testStreams = []
+        url = []
 
-        test = self.testSetup.get("assets").get("device").get(self.testName)
-
-        # Download test artifacts to device
-        url = test.get("artifacts")
-        if url is not None:
-            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
+        streamPaths = self.testSetup.get("assets").get("device").get(self.testName).get("streams")
 
-        url = self.testSetup.get("assets").get("device").get(self.testName).get("streams")
         # Download test streams to device
-        if url is not None:
-            self.downloadToDevice(url, self.deviceDownloadPath, self.rackDevice)
-            for streampath in url:
-                self.testStreams.append(os.path.join(self.deviceDownloadPath, os.path.basename(streampath)))
+        if streamPaths and self.streamDownloadURL:
+            for streamPath in streamPaths:
+                url.append(os.path.join(self.streamDownloadURL, streamPath))
+                self.testStreams.append(os.path.join(self.targetWorkspace, os.path.basename(streamPath)))
+            self.downloadToDevice(url, self.targetWorkspace, self.rackDevice)
 
     def testCleanAssets(self):

```

Comment:
please check this function is required

Reply:
streams not downloaded so removed this function.. removed testExceptionCleanUp() also